test/lib/slim-rails_assets_test.rb

```diff
@@ -28,6 +28,6 @@ class DummyApp < ::Rails::Application
 
   test 'compile slim view' do
     assert_equal 'ok', with_app(false, 'print DummyApp.assets || "ok"')
-    assert_equal '<div class="test">hi</div>', with_app(true, 'print DummyApp.assets["test.slim"].to_s')
```

Comment:
ok i think I got it, the original file in the test is called `test.slim` not `test.html.slim`. I'd add new assertion instead of replacing the older one